tests/Microsoft.DotNet.Framework.Docker.Tests/projects/dotnetapp-3.5/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        Console.WriteLine("Hello Docker World!");
```

Comment:
`Hello Docker World from .NET 3.5` ?

Reply:
Added to all versions to help differentiate apps.